src/main_thread/text_displayer/html/text_track_cues_store.ts

```diff
@@ -234,7 +235,7 @@ export default class TextTrackCuesStore {
     for (let cueIdx = 0; cueIdx < cuesBuffer.length; cueIdx++) {
       let cuesInfos = cuesBuffer[cueIdx];
       if (start < cuesInfos.end) {
-        if (areNearlyEqual(start, cuesInfos.start, relativeDelta)) {
+        if (areCuesStartNearlyEqual(cuesInfosToInsert, cuesInfos)) {
```

Comment:
In that scenario, why is that logic exclusive to the cue's start (and not end for example? Or a mix between the previous's end and the new's start etc.)?

Reply:
In this condition we already know that `start < cuesInfos.end` so we don't need to compare again the previous end ?